samples/tab-deeplink-all-features/csharp/README.md

```diff
@@ -0,0 +1,114 @@
+---
+page_type: sample
+description: Microsoft Teams sample app for demonstrating deeplink all features using Tab
```

Comment:
Microsoft Teams sample app for demonstrating different deep-link features using tab

Reply:
fixed